### ISSUE TITLE
feat: Swipe Controls + Auto HDR

### ISSUE THREAD

**Issue by SahibzadaAliNoor** [revanced/revanced-integrations#105]
### Type

Functionality

### Issue

The aforementioned features have been removed permanently.

### Feature

The were previously available features that made the app better.

### Motivation

Easier to brighten screen without dragging the notification bar or increasing the volume via the physical buttons.

### Additional context

NA

**Comment by Canny1913** [comment]
they are not removed, you can include them using -i

**Comment by TheJeterLP** [comment]
They are not removed permanently. They are just disabled by default. Include them using -i `patchname` when building to apply them.

**Comment by SahibzadaAliNoor** [comment]
Thank you very much. I had been using the GUI to compile the apk and that doesn't allow the -i command. Manual compiling here I come.

**Comment by TheJeterLP** [comment]
> Thank you very much. I had been using the GUI to compile the apk and that doesn't allow the -i command. Manual compiling here I come.

You should open an issue on the GUIs repo then :) As thats not an official tool.

**Comment by SahibzadaAliNoor** [comment]
Yup that's what I was going to do right now 🙊 but I guess I should also learn how to compile it without using a GUI. 

Kind regards,
